tenseal/tensors/bfvvector.cpp

```diff
@@ -27,6 +27,7 @@ BFVVector::BFVVector(const BFVVector& vec) {
 }
 
 size_t BFVVector::size() { return this->_size; }
+size_t BFVVector::ciphertext_size() { return this->ciphertext.size(); }
```

Comment:
Why do we need this call?

Reply:
for unit tests, to measure the ciphertext sizes with/without relin. that was the original issue for this PR